scripts/setup-services.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -e
+
+PG_LOCATION="/usr/lib/postgresql/9.4/bin"
+DB_USERNAME="thimble"
+DB_PASSWORD="thimble"
+ID_WEBMAKER_ENV="POSTGRE_CONNECTION_STRING=postgres://$DB_USERNAME:$DB_PASSWORD@localhost:5432/webmaker_oauth_test HOST=0.0.0.0"
```

Comment:
as a dev setup, hardcoding these should be fine - we already know what they are based on the default configs for login/id